README.md

```diff
@@ -241,6 +241,35 @@ import Countdown, { zeroPad, calcTimeDelta, formatTimeDelta } from 'react-countd
 ### `zeroPad(value, [length = 2])`
 The `zeroPad` function works similar to other well-known pad-functions and takes 2 arguments into account. A `value` which can be a `string` or `number`, as well as a `length` parameter which defaults to `2` as you are most likely only going to use this function if you actually want to pad one of your values. Either returns a `number` if `length` equals `0`, or the zero-padded `string`.
 
+```js
+import React from 'react';
+import ReactDOM from 'react-dom';
+import Countdown, { zeroPad } from 'react-countdown-now';
+
+// Renderer callback with condition
+const renderer = ({ hours, minutes, seconds, completed }) => {
+  if (completed) {
+    // Render a completed state
+    return <span>You are good to go!</span>;
+  } else {
+    // Render a countdown
+    return (
+      <span>
+        {zeroPad(hours)}:{zeroPad(minutes)}:{zeroPad(seconds)}
+      </span>
+    );
+  }
+};
```

Comment:
Hi @keithburgie, thank you for your contribution! 🎉 

Do you think it makes sense to trim these few highlighted lines down to something like this that's more focused on the `zeroPad` function usage rather than how to use conditions there?

```ts
// Renderer callback
const renderer = ({ hours, minutes, seconds }) => (
  <span>
    {zeroPad(hours)}:{zeroPad(minutes)}:{zeroPad(seconds)}
  </span>
);
```

Reply:
I have noticed that this doesn't seem to be as straightforward as I wanted it to be and exposing these functions to the public might lead to more confusion than not doing it. 😁 

But I'm in preparation for updating everything a bit more to make it more clear. There is no need to use these utility functions unless you are motivated and want to calculate time differences yourself for reasons not even I could tell you now to be honest. 😅

Thank you though for your feedback, I appreciate it and will make this last change to this PR unless you want to do it before merging it!

Reply:
I think I updated the PR with your shortened version  ¯\_(ツ)_/¯
This is my first PR on someone else's project. Please let me know if I need to do anything else!